### DISCUSSION TITLE
which carriers are supported?

### DISCUSSION THREAD

**Discussion by rgamage** [karrioapi/karrio#184] [Q&A] [answered]
Looking around at the docs, I couldn't find anywhere, a list of which carriers are supported.  Can someone point me to the right place?  And consider maybe a link in the docs for this?

Thanks,
Randy

**Comment by danh91** [comment, marked as answer]
Hi @rgamage 👋🏾,

Missed this for some reason.

This page of the new documentation site gives you an overview of supported carriers with the integration stage

https://next.purplship.com/docs/carriers
